patch/cisco-Enable-static-memory-reservation-for-OIRable-PCIe-de.patch

```diff
@@ -0,0 +1,198 @@
+commit 9a57f4339767fb8129ee54e9d1714af59d4a6937
+Author: balsup <balsup@contoso.com>
+Date:   Fri Aug 14 00:09:18 2020 +0000
+
+    Reserve address space for powered-off devices behind bridges
+
```

Comment:
Please add the message from the merge/pull request description here, and add Signed-off-by lines. Additionally, please format it with `git format-patch`.

Reply:
Recreated patch with git format-patch process